src/util/caseConv.ts

```diff
@@ -1,3 +1,5 @@
+import { snakeCase } from 'snake-case';
```

Comment:
I don't understand why this needs to use an external library, the current algorithms for snake case conversion are fine.

Reply:
No it's not, it would produce output like this:
![image](https://github.com/PteroPackages/PteroJS/assets/62331820/e8ae9297-5597-4a94-bf09-38f87a416d39)

Reply:
This is a bug caused by `ApplicationServer#toJSON`, environment variables aren't meant to be serialised with these functions:

https://github.com/PteroPackages/PteroJS/blob/0823869d7064ca4f15a2611e4aa977950fa204b9/src/structures/ApplicationServer.ts#L220-L225